doc/helm/user-guide.md

```diff
@@ -61,7 +61,7 @@ If a custom repository URL is specified by `--helm-chart-repo`, the only support
 - `<chartName>`: Fetch the helm chart named `chartName` in the helm chart repository
                  specified by the `--helm-chart-repo` URL.
 
-If `--helm-chart-version` is not set, the SDK will fetch the latest available version of the helm chart. Otherwise, it will fetch the specified version. `--helm-chart-version` is not used when `--helm-chart` itself refers to a specific version, for example when it is a local path or a URL.
+If `--helm-chart-version` is not set, the SDK will fetch the latest available version of the helm chart. Otherwise, it will fetch the specified version. `--helm-chart-version` is not used when `--helm-chart` itself refers to a specific version, for example when it is a local path or a URL. See an example:
 
 Example 
```

Comment:
Note that, the `Example` in the line 66 is not required. 

Also, @joelanford make the following suggestion:

> @alexcpn We already have a number of examples documented in the CLI reference doc. Instead of adding another example here, I think we should link to the existing examples. WDYT?

Then, WDYT about instead of adding the example, address the user to check this doc? 

Such as (add in the line 66)

```
**For more details and examples see the [Helm CLI reference doc][helm-reference-cli-doc]**
```
PS.: The links need to be at the end of the file and then, you can make it work with relative paths. E.g `[operator-scope]:./../operator-scope.md` 

IHMO: it is a better way to address this need since we are allowing the user to check all options and examples for all described in the doc instead of just this specific point.

Reply:
updated. CLI reference page is fine, but I feel a working example (the one I added of a real helm chart url ) helps more than reference. The thing is there are too many frameworks that it is impossible to go to references and learn. That's why people like me go to SO and similar - this could be just one of the many frameworks that one needs to prototype in a days work. Strange, but true. So working examples that generate something, even for people who have never used Helm I feel helps. If you feel otherwise please feel free to decline or edit this the way you want. NP and thanks for the good documentation.